src/CoreWCF.Primitives/src/CoreWCF/Security/XmlHelper.cs

```diff
@@ -411,5 +411,27 @@ public static UniqueId ReadTextElementAsUniqueId(XmlElement element)
         {
             return new UniqueId(ReadTextElementAsTrimmedString(element));
         }
+
+        internal static string[] TokenizeInclusiveNamespacesPrefixList(string inclusiveNamespacesPrefixList)
+        {
+            if (inclusiveNamespacesPrefixList == null)
+                return null;
+
+            string[] prefixes = inclusiveNamespacesPrefixList.Split(new char[] { ' ' }, StringSplitOptions.RemoveEmptyEntries);
```

Comment:
You can pass `null` as the first parameter. The "The separator array" section of the documentation says:

> If the separator parameter is null or contains no characters, white-space characters are assumed to be the delimiters.

You could also pass `Array.Empty<char>` if there's problems with ambiguous overloads.
```suggestion
            string[] prefixes = inclusiveNamespacesPrefixList.Split(null, StringSplitOptions.RemoveEmptyEntries);
```